.github/workflows/python-test.yml

```diff
@@ -0,0 +1,39 @@
+name: Test Python Package
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6]
```

Comment:
should we have more python versions here?

Reply:
I want to add 3.6 - 3.9.